src/main/java/com/Elessar/database/MyDatabase.java

```diff
@@ -0,0 +1,19 @@
+package com.Elessar.database;
```

Comment:
Is Document MongoDB specific type? I thought it was independent since it is from `org.bson.Document`

Reply:
For find operation, why the function is named as `findUsers` not simply`find`? Is it to differentiate from other methods since find may possible match multiple users while other method will only match single user